virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -6,33 +6,25 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+{VirtProductName} is installed with a default internal pod network.
 
-To attach a virtual machine to an additional network:
+Depending on the type of networks you are using, it may be necessary to update your network node configuration.
 
-. Configure a Linux bridge.
-. Configure a bridge network attachment definition for a namespace in the web console or CLI.
-+
-[NOTE]
-====
-The network attachment definition must be in the same namespace as the pod or virtual machine.
-====
-. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
-** In the web console, create a NIC for a new or existing virtual machine.
-** In the CLI, include the network information in the virtual machine configuration.
+In order to use your virtual machines with external networks, you must create a Linux _network bridge_ to expose your network services outside the default internal pod network.
 
-[NOTE]
-====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
-====
+Once a network bridge is created, you can start virtual machines on the external network in the same manner as you start virtual machines attached to internal networks, by using PXE booting, for example.
```

Comment:
"you can start virtual machines on the external network in the same manner as you start virtual machines attached to internal networks, by using PXE booting, for example." 

This is a use case, so it should probably go earlier in the assembly. I think you will need something like this: 

{VirtProductName} is installed with a default internal `pod` network. If you want to attach VMs to an external network, for example, to support PXE booting, you must create a network bridge.

Reply:
Agreed, better. On 'pod` should we refer to the object or just "pod" in general?

Reply:
I'll use the ticks for now